services/spar/src/Spar/Intra/Brig.hs

```diff
@@ -169,6 +170,10 @@ createBrigUser suid (Id buid) teamid mbName managedBy = do
       | otherwise ->
         throwSpar . SparBrigError . cs $ "create user failed with status " <> show sCode
 
+updateEmail :: (HasCallStack, MonadSparToBrig m) => UserId -> Email -> m ()
```

Comment:
We should not introduce any new hacks that call `/self` endpoints with forged authorization headers from other services. Instead add a proper internal endpoint instead. This makes our lives hard when implementing federation; and is also just hacky

Reply:
ok, fair point.  i'll add an internal end-point that does exactly the same thing, it's just a few lines of boilerplate.  thanks for reminding me!

Reply:
there is actually a `put /i/self/email`, which only needs to be augmented with a query flag to allow for sending out emails.